runtime/opal/src/lib.rs

```diff
@@ -1092,7 +1092,6 @@ type SponsorshipHandler = (
 
 impl pallet_evm_transaction_payment::Config for Runtime {
 	type EvmSponsorshipHandler = EvmSponsorshipHandler;
-	type Currency = Balances;
```

Comment:
In the future separate these entries into "code cleanup" commits/PRs to make PRs more readable.